src/encoded/static/components/blocks/richtext.js

```diff
@@ -58,32 +57,46 @@ export default class RichTextBlockView extends React.Component {
         });
     }
 
+    // Clickable FAQs: hide and show answers corresponding to questions
     handleClickableFAQ() {
-        document.getElementsByClassName("faq")[0].addEventListener("click", function(e) {
-            let target = null;
-            for (let i = 0; i < e.path.length; i++) {
-                let tempPath = e.path[i].className;
-                if (tempPath){
-                   if (tempPath.indexOf("regulomehelp-question") !== -1) {
-                        target = e.path[i];
-                        break;
+        if (this.state.value.body.includes('faq')) {
+            const faqContainer = document.getElementsByClassName('faq')[0];
+            if (faqContainer) {
+                // One event listener on the 'faq' container handles all click events
+                faqContainer.addEventListener('click', (e) => {
+                    // Polyfill for 'closest' for IE
+                    const elementPrototype = window.Element.prototype;
+                    if (!elementPrototype.closest) {
```

Comment:
This test for `closest` struck me as a little bit odd because this test would happen every time you click. But I think it’s OK since the performance penalty would be probably unmeasurable. But I think it’s more expected to test and assign the polyfill in componentDidMount().

Reply:
Yes, that would be better. I was actually sort of wondering if it should be in objectutils.js or app.js. If we wanted to use `closest` anywhere else it wouldn't make sense to keep it here.

Reply:
Maybe even browserfeat.js? That only tests for features now, but it could add features.